libretroshare/src/rsserver/rsinit.cc

```diff
@@ -1929,3 +1934,88 @@ int RsServer::StartupRetroShare()
 	return 1;
 }
 
+RsInit::LoadCertificateStatus RsLoginHelper::attemptLogin(
+        const RsPeerId& account, const std::string& password)
+{
+	if(!rsNotify->cachePgpPassphrase(password)) return RsInit::ERR_UNKOWN;
+	if(!rsNotify->setDisableAskPassword(true)) return RsInit::ERR_UNKOWN;
+	if(!RsAccounts::SelectAccount(account)) return RsInit::ERR_UNKOWN;
+	std::string ignore;
+	RsInit::LoadCertificateStatus ret =
+	        RsInit::LockAndLoadCertificates(false, ignore);
+	rsNotify->setDisableAskPassword(false);
+	if(ret != RsInit::OK) return ret;
+	if(RsControl::instance()->StartupRetroShare() == 1) return RsInit::OK;
+	return RsInit::ERR_UNKOWN;
```

Comment:
It seems to me we should clear the cached password here right after LockAndLoadCertificates(), since it was passed as a parameter.

Reply:
Done! Thanks!